### ISSUE TITLE
Add image with title and alt text to concept module

### ISSUE THREAD

**Issue by oraNod** [redhat-documentation/modular-docs#172]
During recent review with Lorena and Ingrid, I got feedback about adding a title and alt text to images. Specific feedback comment: "This diagram also needs a name and title."

If my asciidoc source is `image::image.png[My image]` then what markup should I add? 

I can go refer to the asciidoc user's manual and find the tags I need but I'd rather have that in the template for concept modules as there should be guidelines.

**Comment by babysophia1960** [comment]
Really Lori what the fuck are you doing your going to get fucked if you keep doing this shit I swear to God I'm gonna put Marcus in jail￼

**Comment by sterobin** [comment]
@oraNod I've reported the user of the above content to GitHub and have hidden the comment since it's highly offensive and concerning. I chose not to delete it in case it's needed for record. I also blocked the user, but couldn't tell if that blocked them just from my repositories or from mod docs altogether. Seems like the block option is only for repos that I own, my forks.

**Comment by emmurphy1** [comment]
@adahms, we added some instructions about images and alt text to the concept module template.
